eslint-local-rules/utils.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @param {String} filename
+ * @returns {Boolean}
+ */
+function isReactViewFile(filename) {
+    return filename.includes('/src/pages/') || filename.includes('/src/components');
```

Comment:
I wonder if it would be better or more consistent to only allow Onyx to be referenced from `src/lib/actions`? 🤔

Reply:
Good point. We do say that "actions are the only thing that should set data". So probably a good thing to enforce.

This rule is currently just looking for "Onyx" usages but we could split this into:

- `set()`, `merge()`, `mergeCollection()` etc should always be called from `/lib/actions` files
- `Onyx.connect()` should only appear in `/libs/**`

Reply:
Yeah, I like that. There are quite a few usages of `Onyx.connect()` outside of `actions` which seems like it is OK.